dali/pipeline/operators/resize/resize.cc

```diff
@@ -20,20 +20,32 @@ namespace dali {
 DALI_SCHEMA(ResizeAttr)
   .AddOptionalArg("image_type",
         R"code(The color space of input and output image.)code", DALI_RGB)
-  .AddOptionalArg("resize_x", "The length of the X dimension of the resized image. "
-      "This option is mutually exclusive with `resize_shorter`. "
-      "If the `resize_y` is left at 0, then the op will keep "
-      "the aspect ratio of the original image.", 0.f, true)
-  .AddOptionalArg("resize_y", "The length of the Y dimension of the resized image. "
-      "This option is mutually exclusive with `resize_shorter`. "
-      "If the `resize_x` is left at 0, then the op will keep "
-      "the aspect ratio of the original image.", 0.f, true)
-  .AddOptionalArg("resize_shorter", "The length of the shorter dimension of the resized image. "
-      "This option is mutually exclusive with `resize_longer`, `resize_x` and `resize_y`. "
-      "The op will keep the aspect ratio of the original image.", 0.f, true)
-  .AddOptionalArg("resize_longer", "The length of the longer dimension of the resized image. "
-      "This option is mutually exclusive with `resize_shorter`,`resize_x` and `resize_y`. "
-      "The op will keep the aspect ratio of the original image.", 0.f, true);
+  .AddOptionalArg("resize_x", R"code("The length of the X dimension of the resized image.
+      This option is mutually exclusive with `resize_shorter`.
```

Comment:
Are you sure if those spaces between lines play well in Sphinx docs?

Reply:
No, you are right, there shouldn't be space here

Reply:
Fixed